testing/snapshot.ts

```diff
@@ -124,7 +129,8 @@
  * });
  * ```
  *
- * ```js
+ * @example
+ * ```ts
  * // .snaps/example_test.ts.snap
```

Comment:
When changed to a `ts` block, types fail. Should this be `.snaps/example_test.js.snap`?